### PR TITLE
Added entries to changelog concerning 2019.10 release

### PR DESCRIPTION
**Pull request by blattms**
Please review carefully. Will be in the release, too.

### CODE REVIEW

**Comment by akva2** [comment]
we should not announce the python stuff, it will not be enabled in this release.

File: changelog.md [hunk 1]
```diff
@@ -3,6 +3,26 @@
 A short month-by-month synopsis of change highlights. Most bugfixes won't make
 it in here, only the bigger features and interface changes.
 
+# Important changes between release 2019.04 and 2019.10
+
+* opm-common and the rest of OPM does not use libecl anymore and
+  supports reading and writing Eclipse files directly
+* improved Eclipse compatible restart support (including e.g. UDQ
```
Comment (joakim-hove) [inline changelog.md]: The UDQ and ACTIONX support is not really complete when it comes to restart.

Comment (blattms) [reply changelog.md]: then please rephrase.

Comment (joakim-hove) [reply changelog.md]: I would just remove the full bullet point.

**Comment by blattms** [comment]
backported to release via #1183

**Pull request merged**